File: Exercicios não concluidos ou errados/apartamento.py
I = float(input())
A = float(input())
T = float(input())

if  I>0 and A>0:
    if I == A:
        print('pode comprar agora')
    else:
        meses = 0
        while I <= A:
            I += I*T/100
            meses += 1
        if meses > 1:
            print(f'possivel em {meses} meses')
        else:
            print(f'possivel em {meses} mes')
        

File: apartamento.cpp
#include <iostream>
#include <math.h>

main() {
    int cont;
    double I, A, T;
    bool p = true;

    scanf("%lf", &I);
    scanf("%lf", &A);
    scanf("%lf", &T);

    if (I >= A){
        printf("pode comprar agora");
    }
    else{
        while (p == true){
            I = I + I*T/100;
            cont ++;
            if (I >= A){
                p = false;
            }
        }
        if (cont > 1){
            printf("possivel em %d meses", cont);
        }
        else{
            printf("possivel em %d mes", cont);
        }

    }

    
}
